src/khoj/utils/helpers.py

```diff
@@ -184,6 +184,7 @@ def log_telemetry(telemetry_type: str, api: str = None, client: str = None, app_
     request_body = {
         "telemetry_type": telemetry_type,
         "server_id": get_server_id(),
+        "server_version": version("khoj-assistant"),
         "os": platform.system(),
```

Comment:
Does the telemetry server require any schema changes to accept the new value?

Reply:
Ah, nice catch! Yes, the table schema in sqlite db would need to be updated

Reply:
1. Manually added a `server_version` column to the new telemetry server
2. Update the telemetry server code to get and push server_version column from API to sqlite3 db when server_version available in request payload
3. Updated the deployed version of the telemetry server to use this new server code